<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>295</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Portable/Resources/logo1.png</normaloff>:/Portable/Resources/logo1.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="cpc_pushbutton" name="objadvcfg">
       <property name="text">
        <string>高级&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="cpc_pushbutton" name="objlogin">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="cpc_pushbutton" name="objquit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="cpc_label" name="objLogo">
     <property name="minimumSize">
      <size>
       <width>423</width>
       <height>115</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="cpc_label" name="label">
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="cpc_label" name="label_2">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="cpc_lineedit" name="objpsw">
       <property name="text">
        <string>cpc123</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="objautologin">
       <property name="text">
        <string>自动登录</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="objaccount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="3">
         <widget class="cpc_pushbutton" name="objtest">
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="cpc_label" name="label_3">
          <property name="text">
           <string>服务端IP：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="cpc_label" name="label_5">
          <property name="text">
           <string>本机IP：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="cpc_lineedit" name="objServerip">
          <property name="text">
           <string>192.168.1.75</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="cpc_combobox" name="objlocalip"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="cpc_label" name="objLoginStatus">
     <property name="text">
      <string>未登录，请先&lt;连接&gt;再登录...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cpc_pushbutton</class>
   <extends>QPushButton</extends>
   <header location="global">cpc_pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>cpc_label</class>
   <extends>QLabel</extends>
   <header location="global">cpc_label.h</header>
  </customwidget>
  <customwidget>
   <class>cpc_lineedit</class>
   <extends>QLineEdit</extends>
   <header location="global">cpc_lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>cpc_combobox</class>
   <extends>QComboBox</extends>
   <header location="global">cpc_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IPCTestDemo.qrc"/>
 </resources>
 <connections/>
</ui>
